### ISSUE TITLE
OgdenElastic material model - mu field not set

### ISSUE THREAD

**Issue by iBatistic** [solids4foam/solids4foam#140]
When using `OgdenElastic` material model, the following error shows:
```
--> FOAM FATAL ERROR: 
mu field has not been set. Please construct the field first using mu(const dimensionedScalar& mu)

    From function Foam::volScalarField& Foam::mechanicalLaw::mu()
    in file materialModels/mechanicalModel/mechanicalLaws/mechanicalLaw/mechanicalLaw.C at line 549.
```
I will check it and propose PR for fix.

**Comment by iBatistic** [comment]
@philipcardiff `YeohElastic`, `OgdenElastic`, and `isotropicFungElastic` material models encountered the same error, which was resolved with commit #d51b696 (commit is within PR #138). The issue originated from a fix implemented in PR #90, after which the models mentioned were not updated accordingly.

**Comment by philipcardiff** [comment]
Fixed
